packages/gatsby-theme-newrelic/gatsby/type-defs.graphql

```diff
@@ -52,3 +53,13 @@ type NewRelicThemeTessenConfig {
   product: String
   subproduct: String
 }
+
+type googleTagManagerConfig {
+  trackingId: String
+  src: String
+  options: googleTagManagerOptions
+}
+
+type googleTagManagerOptions {
```

Comment:
```suggestion
type GoogleTagManagerConfig {
  trackingId: String
  src: String
  options: GoogleTagManagerOptions
}

type GoogleTagManagerOptions {
```

GraphQL types are standardized around capital names to denote them as types rather than variables.

Reply:
not sure how i missed that 🤦  thanks for catching that!